### PR TITLE
KL div loss: add kldiv_loss op

### PR DESCRIPTION
**Pull request by heavengate**
https://pytorch.org/docs/master/nn.html#torch.nn.KLDivLoss

### CODE REVIEW

**Comment by heavengate** [comment]
@haowang101779990 please check, thanks

**Comment by haowang101779990** [comment]
The updated API contents LGTM，the thumbnail of API at the top is outdated, please update it in time. 👍  @shanyi15

**Comment by shanyi15** [comment]
image

**Comment by heavengate** [comment]
Done, please check @shanyi15

**Review by shanyi15 — APPROVED** [review]
LGTM, please ask another RD to check the example in doc

**Review by phlrain — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
